backends/advanced/notebooks/debugmemory.ipynb

```
@@ -0,0 +1,204 @@
+   "execution_count": 1,
+   "id": "412d0a88-a015-4c44-99f8-4f03feba1124",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
```

Ruff (0.12.2)

`await` statement outside of a function

(F704)